workspaces/local-cli/src/commands/testing/enable.ts

```diff
@@ -0,0 +1,56 @@
+import Command from '@oclif/command';
+import colors from 'colors';
+import fs from 'fs-extra';
+import {
+  developerDebugLogger,
+  loadPathsAndConfig,
+  fromOptic,
+  promiseFromOptic,
+  SaasClient,
+} from '@useoptic/cli-shared';
+import { cli } from 'cli-ux';
+import UrlJoin from 'url-join';
+import Config from '../../config';
+
+export default class Enable extends Command {
+  static description = 'Enable Live Contracting Testing for your API';
+
+  async run() {
+    const { paths, config } = await loadPathsAndConfig(this);
+
+    if (fs.existsSync(paths.testingConfigPath)) {
+      return this.log(
+        fromOptic(
+          'Testing is already enabled for this API. To re-enable, use testing:disable command first.'
+        )
+      );
+    }
+
+    const baseUrl = UrlJoin(Config.apiGatewayUrl, 'api/v1');
+    const saasClient = new SaasClient(baseUrl);
+    developerDebugLogger('fetching auth token');
+
+    const gettingApiToken = saasClient
+      .getApiAuthToken(config.name || 'Unknown API')
+      .catch(() => {
+        cli.action.stop;
```

Comment:
seems like we're missing a cli.action.start() before this, and missing the invocation here? Also, are we using both this and ora for indeterminate?

Reply:
Just a bit of code I hadn't removed yet 😅